packages/vitest/src/node/core.ts

```diff
@@ -688,11 +693,28 @@ export class Vitest {
     await this.report('onWatcherStart', this.state.getFiles(files))
   }
 
+  private isSuite(task: RunnerTask): task is RunnerTestSuite {
+    return Object.hasOwnProperty.call(task, 'tasks')
+  }
+
+  async rerunTask(id: string) {
+    const task = this.state.idMap.get(id)
+    if (task) {
```

Comment:
maybe we should throw an error if task it not found?

Reply:
yeah, I'll ask you about this... or maybe we can rerun all files or passing the filename back again and rerun all files

Reply:
I think an error is better